docs/2.8. Usage - Testing Controllers.md

```diff
@@ -2,9 +2,10 @@
 
 ## Running the Semi-automatic Controller Tests
 
-Launch the Testing Façade from the testingfacade directory. By default the Testing Façade will run on port 5001
+Launch the Testing Façade. By default the Testing Façade will run on localhost port 5001. This can be altered by adding `TESTING_FACADE_HOST` and `TESTING_FACADE_PORT` to your `UserConfig.py` file as described in [1.0 Installation](1.0.%20Installation.md).
```

Comment:
I thought it would still be reasonable to allow connections on any interface, using `0.0.0.0`, so I expected only `TESTING_FACADE_PORT` not ...`_HOST` too?

Reply:
Ok I'll take `HOST` out. Just thought it made sense to have both